components/Slider/examples/SliderCollectionViewController.m

```diff
@@ -76,7 +76,7 @@ - (void)didChangeMDCSliderValue:(MDCSlider *)slider {
 
 @interface MDCSliderExampleCollectionViewCell : UICollectionViewCell
 - (void)applyModel:(MDCSliderModel *)model withColorScheme:(MDCSemanticColorScheme *)colorScheme;
-
+@property (nonatomic, strong, nullable) UIFont *labelFont;
```

Comment:
Nit/style: Please place properties at the top of the `@interface`

Reply:
Done